pages/api/user.ts

```diff
@@ -0,0 +1,17 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+
+let cache: {
```

Comment:
The server for next js ran globally so this would be overwritten by other users.